### PR TITLE
Minimal support for PVR Addon API 1.9.7

### PR DESCRIPTION
**Pull request by ksooo**
For, J**** PVR Addon API will be bumped to version 1.9.7 to introduce a couple of new features and enhancements for timers (esp. official series recording support). => https://github.com/xbmc/xbmc/pull/7079

This PR will ensure compatibility with and add minimalistic support for PVR addon API 1.9.7. Even if the maintainers of this addon will do nothing except merging this PR and bump the addon version in Kodi, the addon will still be usable in J**** and not loose any functionality compared to Isengard, but it will not profit from the new features. For the latter, every addon maintainer must do the respective changes on his own.

### CODE REVIEW

File: src/client.cpp [hunk 1]
```diff
@@ -305,7 +305,6 @@ extern "C" {
     pCapabilities->bSupportsChannelScan = false;
     pCapabilities->bSupportsChannelSettings = false;
     pCapabilities->bHandlesDemuxing = false;
-    pCapabilities->bSupportsRecordingFolders = false;
```
Comment (Jalle19) [inline src/client.cpp]: Why did you remove this?

Comment (ksooo) [inline src/client.cpp]: Because this is now a timer type attribute. Each timer type can define whether folders are supported. For example, tvheadend only supports recording folders for autorec and timerec, but not for one-shot timers. Thus, defining this an addon capability was, well ..., not quite correct, but the best solution at that time. Now we can do better.

Comment (ksooo) [inline src/client.cpp]: ... consequently, I removed the capabiliy from the API (and Kodi implementation).

Comment (Jalle19) [inline src/client.cpp]: I see, thanks for the clarification!
